api-gateway/caching-api-response/api-cache.yml

```diff
@@ -9,10 +9,13 @@ services:
     image: rodolpheche/wiremock
     volumes:
       - "../../common-services/webapi:/home/wiremock"
+      # Remember to put https://repo1.maven.org/maven2/com/opentable/wiremock-body-transformer/1.1.3/wiremock-body-transformer-1.1.3.jar in ../../common-services/webapi/extensions directory!
```

Comment:
Can we automate it with Gradle? This jar file should be ignored by Git.

Reply:
done